shell/machine-config/__tests__/vmwarevsphere.test.ts

```diff
@@ -1,6 +1,6 @@
 import { mount } from '@vue/test-utils';
 import vmwarevsphere, { DEFAULT_VALUES, SENTINEL } from '@shell/machine-config/vmwarevsphere.vue';
-import { cleanHtmlDirective } from '@shell/plugins/clean-html-directive';
+import cleanHtmlDirective from '@shell/plugins/clean-html-directive';
```

Comment:
We might be able to remove these imports from our individual test files and register the plugin in `jest.setup.js`.